.cirrus.yml

```diff
@@ -315,329 +187,6 @@ image_prune_task:
     script: '/usr/local/bin/entrypoint.sh |& ${TIMESTAMP}'
 
```

Comment:
Please nuke image_prune as well.  It only ever runs on the master branch...and is currently disabled anyway.